src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationSource.cs

```diff
@@ -23,6 +25,19 @@ public AzureAppConfigurationSource(Action<AzureAppConfigurationOptions> optionsI
 
         public IConfigurationProvider Build(IConfigurationBuilder builder)
         {
+            try
+            {
+                if (Environment.GetEnvironmentVariable(ConditionalProviderConstants.DisableProviderEnvironmentVariable) == "true")
```

Comment:
This would require case insensitive comparison. Read the env var as a string and then use `bool.TryParse` to get the value.

Reply:
Sample here: https://github.com/Azure/AppConfiguration-DotnetProvider/blob/9a76d2a1b43d137e139e329a575eee09641e51d3/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs#L115